python/lsst/pipe/supertask/activator.py

```diff
@@ -292,7 +321,11 @@ def load_super_task(super_taskname):
 
         if super_task_module:
             # Need to add a try -- except here
-            python_mod_stask = importlib.import_module(package.__name__ + '.' + super_task_module)
+            if hasattr(package, 'mock'):
+                (file_path, filename, data) = imp.find_module(super_task_module, package.__path__)
+                python_mod_stask = imp.load_module(super_task_module, file_path, filename, data)
```

Comment:
Can't you add `package.__path__` to `sys.path` and then just use `importlib.import_module`? (and then return `sys.path` to it's initial state).